ckan/lib/webassets_tools.py

```diff
@@ -1,172 +1,244 @@
-# encoding: utf-8
 from __future__ import annotations
 
 import logging
 import os
 import tempfile
 from typing import Any, Optional
+from typing_extensions import Literal, TypedDict, assert_never
 
-import yaml
 from markupsafe import Markup
 from webassets import Environment
 from webassets.loaders import YAMLLoader
 
 from ckan.common import config, g
 
 
-logger = logging.getLogger(__name__)
+log = logging.getLogger(__name__)
 env: Optional[Environment] = None
 
-yaml.warnings({u'YAMLLoadWarning': False})
+AssetType = Literal["style", "script"]
+
+
+class AssetCollection(TypedDict):
+    script: list[str]
+    style: list[str]
+    included: set[str]
 
 
 def create_library(name: str, path: str) -> None:
     """Create WebAssets library(set of Bundles).
     """
-    config_path = os.path.join(path, u'webassets.yml')
+    if not env:
+        log.debug("Webassets environment is not initialized yet")
```

Comment:
Shouldn't we increase the logging level to CRITICAL? If Webassets environment is not initialized then this means that something important is not gonna we working.